pos-print/src/main/java/com/clouway/pos/print/printer/FP705Printer.java

```diff
@@ -173,11 +169,18 @@ public PrintReceiptResponse printFiscalReceipt(Receipt receipt) throws IOExcepti
       channel.sendPacket(buildPacket(seq++, FISCAL_RECEIPT_PRINT_TEXT, params(prefix)));
     }
 
+
     double sum = 0;
     for (ReceiptItem item : receipt.getReceiptItems()) {
+      int vat = DEFAULT_VAT_GROUP;
+      for (FiscalPolicy policy : fiscalPolicy) {
+        if (policy.getVat() == item.getVat()) {
+          vat = policy.getGroup();
```

Comment:
Group should be string and not an int as some cash registers may use chars: "A", "B", "C".